apps/mdn/mdn-aws/infra/mdn-cdn/cloudfront_attachments/cloudfront_attachments.tf

```diff
@@ -0,0 +1,72 @@
+variable "distribution_name" {}
+variable "comment" {}
+variable "domain_name" {}
+variable "acm_cert_arn" {}
+
+variable "aliases" {
+  type = "list"
+}
+
+resource "aws_cloudfront_distribution" "mdn-attachments-cf-dist" {
+  aliases         = "${var.aliases}"
+  comment         = "${var.comment}"
+  enabled         = true
+  http_version    = "http2"
+  is_ipv6_enabled = false
+  price_class     = "PriceClass_All"
+
+  custom_error_response {
+    error_caching_min_ttl = 300
+    error_code            = 404
+  }
+
+  default_cache_behavior {
+    allowed_methods = ["GET", "HEAD", "OPTIONS"]
+    cached_methods  = ["GET", "HEAD", "OPTIONS"]
+    compress        = true
+
+    default_ttl     = 86400
+    max_ttl         = 31536000
```

Comment:
In https://github.com/mozilla/kuma/pull/4476, we're setting the ``Cache-Control`` headers for most responses, so [according to the docs](http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html) we'll get the lesser of ``max_ttl`` and the ``max-age`` we're sending.  For some we're sending 5 minutes (60 * 5 = 300), others 5 days (60 * 60 * 24 * 5 = 432000). I think we're not sending caching headers on some error responses.

I'm more comfortable with 24 hours for default (86400, current setting), and 5 days for max (432000). 1 year seems like asking for trouble for the next devs.

Reply:
fixed in the latest push.

btw, the 1 year value came from the SCL3 Cloudfront config :-)

Reply:
😳